lua/src/lib/library.lua

```diff
@@ -20,6 +17,11 @@ M.split = function(str, delim)
   end
   return _accum_0
 end
+M.replace = function(str, what, sub)
+  what = string.gsub(what, "[%(%)%.%+%-%*%?%[%]%^%$%%]", "%%%1")
```

Comment:
where did you come up with this list of characters? Is there a lua doc somewhere? If so, can we add a comment to the code?